BookLibrary-cdi/src/main/java/org/booklibrary/app/web/controller/AuthorController.java

```diff
@@ -0,0 +1,84 @@
+package org.booklibrary.app.web.controller;
+
+import org.booklibrary.app.manager.AuthorManagerLocal;
+import org.booklibrary.app.manager.exceptions.EntityManagerException;
+import org.booklibrary.app.persistence.entity.Author;
+import org.slf4j.Logger;
+
+import javax.enterprise.inject.Produces;
+import javax.faces.view.ViewScoped;
+import javax.inject.Inject;
+import javax.inject.Named;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+@Named
+@ViewScoped
+public class AuthorController implements Serializable{
+
+    @Inject
+    private Logger LOG;
+
+    @Inject
+    private AuthorManagerLocal authorManager;
+
+    private Author managedAuthor = null;
+    private List<Author> authors = new ArrayList<>();
+    private int page = 1;
+
+    public void save() {
+        if (managedAuthor != null) {
+            try {
+                authorManager.save(managedAuthor);
+            } catch (EntityManagerException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    public void remove() {
+        if (managedAuthor != null) {
+            try {
+                authorManager.removeByPk(managedAuthor.getId());
+            } catch (Exception e) {
+                e.printStackTrace();
```

Comment:
please handle exception either by throwing it up or logging in. Do not use e.printStackTrace() http://stackoverflow.com/questions/7469316/why-is-exception-printstacktrace-considered-bad-practice